### DISCUSSION TITLE
Searching for "generic" elemental res on gear

### DISCUSSION THREAD

**Discussion by makuta2** [Sidekick-Poe/Sidekick#284] [Ideas]
When searching for elemental res on gear, could we get an option to "merge" all 3 res into one "generic elemental res"
for example, a pair of gloves has 20% fire res and 30% cold res, the option would be 50% elemental res
and the search parameters would be under "weighted sum", where fire, cold, and lightning res are weighted as 1, with a collective minimum weight as 50.

in poe1, this was covered by the trade website's "pseudo elemental res" search function, but poe2 does not have that trade feature yet.

**Comment by leMicin** [comment]
This is supported in PoE1 through Pseudo modifiers that are available on the trade site.
PoE2 trade site does not have pseudo modifiers yet. When GGG adds them, we will be quick to add them ourselves.

If pseudo modifiers are never a thing, then this approach could work. But how would we do the weighted sum with the All elemental resistance modifier? Technically that should count as 3 separate resistances of Fire, Cold, Lightning. It`s interesting to think about. I may need to dig into weighted sum some more.

**Reply by makuta2** [reply]
![image](https://github.com/user-attachments/assets/cf53d3a4-094f-4b9d-8472-94e432774ac1)
you would weight fire, cold, lightning into one section under "weighted sum"
then set the values similar to the feature of "less than", "more than", "equal to", or "min-max range"

**Reply by makuta2** [reply]
once you figure out how to do that, it would be easy to apply it to other sets of elemental modifiers such as added elemental damage, increased elemental damage, etc.
Of course only if poe2 never adds it to their trade website
![image](https://github.com/user-attachments/assets/6e385140-0430-434c-9f24-96a5c6423211)

**Reply by leMicin** [reply]
And for all res, we do a weighted sum of 3 instead of 1?

**Reply by makuta2** [reply]
yes that sounds good

**Reply by makuta2** [reply]
I'll make a list of "elemental" modifiers that could also be good candidates for this feature:
"Gain #% of Damage as Extra Elemental Damage"
"Add # to # Elemental Damage"
"Add # to # Elemental Damage to Attacks"
"#% Increased Elemental Damage"

**Reply by makuta2** [reply]
I had to double check, hence previous comment's edit
![image](https://github.com/user-attachments/assets/4fb4b0ed-769b-4ff7-b805-8199110d3e3f)
Implicit and normal res need to be added separately.

**Reply by gh0st0ff0x** [reply]
> And for all res, we do a weighted sum of 3 instead of 1?

Actually, [this not easy request](https://www.pathofexile.com/trade2/search/poe2/Standard/KGbd8XLh5) is to look any 25% resistances on armor pieces. But API can't hold it, I'm afraid, because of length.

**Reply by gh0st0ff0x** [reply]
If item has 50% res in sum, request should be like [this](https://www.pathofexile.com/trade2/search/poe2/Standard/VXlvwPoCp)

**Reply by gh0st0ff0x** [reply]
Oh, of course, if you want to include "+#% to all Elemental Resistance" just add it and point weight to 3 (because 10% all RES*3 equal to 30% fire RES, for example)

**Reply by gh0st0ff0x** [reply]
> "Add # to # Elemental Damage to Attacks"

More complicated than elemental resistances.
Firs of all it should be summed like: 1st(MIN)+2nd(MAX)/2. And then pointed in min input on the trade site.
"Add to 1 to 29 lightning damage to attacks" equal to "Add to 10 to 20 physical damage to attacks" in average.

I hope it will help.

**Reply by leMicin** [reply]
We have the feature ready... #417 
Unfortunately, we can't enable it yet due to high restrictions on complex queries. We need to ask for the user authentication, or wait for GGG to allow weighted sums in the anonymized API.

Right now work is paused on pseudo modifiers as we have more issues to fix.
